### DISCUSSION TITLE
Does `stagger` not work with `animateMini`?

### DISCUSSION THREAD

**Discussion by brandonpittman** [motiondivision/motion#2866] [Q&A] [answered]
This snippet works:

```ts
    const animation = animate(
      "[data-loading-dot]",
      {
        backgroundColor: ["var(--color-mid-glare)", "var(--color-primary)"],
        transform: ["scale(1)", "scale(1.25)", "scale(1)"],
      },
      {
        delay: stagger(1),
        duration: 1,
        ease: easeInOut,
      }
    );
```

If I switch to `animateMini` I get a `TypeError: The provided value is non-finite.`

The docs say `stagger` works with `animateMini`, but this error happens with the mini function and goes away if I remove `stagger`.

**Comment by mattgperry** [comment]
It should but I’ll take a look!

**Comment by mattgperry** [comment, marked as answer]
Fix incoming https://github.com/motiondivision/motion/pull/2877

**Comment by brandonpittman** [comment]
Stagger works again!
